### PR TITLE
mofidy demo to work with different dpr

### PR DESCRIPTION
**Pull request by loonybear**
Hi Eric,

I added 3 cases with different device pixel ratio, please take a look. thanks

### CODE REVIEW

File: public/demos/max-downscaling-image.html [hunk 1]
```diff
@@ -34,39 +34,115 @@ <h2 id="feature-allowed-banner">
 -->
 
 <section class="layout center wrap" style="justify-content:space-evenly;">
+  <h3>
+    When device pixel ratio is 1 (e.g. tablets, microsoft surface, etc)
+  </h3>
   <div class="layout vertical center-center">
-    <img src="../img/Fisher-large-stripped.jpg" width="200" height="150">
+    <img src="../img/emilio-large.jpg" width="800" height="500">
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: This is kinda confusing b/c the image you're using for dpr 1 and 2 are the same and the DPR isn't changing really, it's the size you're scaling it by is in the demo. I think you should update the `h3` titles to describe that instead and under what scenarios the demos will fail. Does that make sense?

**Comment by loonybear** [comment]
Done. Please take another look. Thanks

File: public/demos/max-downscaling-image.html [hunk 2]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
+    For devices with device pixel ratio of 1 (e.g. tablets, monitors, etc), the compression ratio (800/380) > 2, image renders with inverted clolors.
+    </p>
+  </div>
+
+  <div class="layout vertical center-center">
+    <img src="../img/emilio-large.jpg" width="180" height="500">
+    <p>
+    Intrinsic size: 800px x 500px <br />
+    Container size: 180px x 500px
+    For devices with device pixel ratio of 2 (e.g. iPad, MacBook, smartphones, etc), the compression ratio (800/(180 * 2)) > 2, image renders with inverted clolors.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: clolors -> colors

File: public/demos/max-downscaling-image.html [hunk 3]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
 -->
 
 <section class="layout center wrap" style="justify-content:space-evenly;">
+  <h3>
+    When the compression ratio is less than or equal to maximum downscaling ratio (2),
+    image renders normally.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: image -> the image

File: public/demos/max-downscaling-image.html [hunk 4]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
-    Intrinsic size: 200px x 150px <br />
-    Container size: 200px x 150px
+    Intrinsic size: 800px x 500px <br />
+    Container size: 800px x 500px
+    </p>
+  </div>
+</section>
+<section class="layout center wrap" style="justify-content:space-evenly;">
+  <h3>
+    When the compression ratio (width) is more than maximum downscaling raio, image renders
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: image renders -> the image renders

File: public/demos/max-downscaling-image.html [hunk 5]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
+    with inverted colors.
+  </h3>
+  <div class="layout vertical center-center">
+    <img src="../img/emilio-large.jpg" width="380" height="500">
+    <p>
+    Intrinsic size: 800px x 500px <br />
+    Container size: 380px x 500px
+    For devices with device pixel ratio of 1 (e.g. tablets, monitors, etc), the compression ratio (800/380) > 2, image renders with inverted clolors.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: image renders -> the image renders

Comment (ebidel) [inline public/demos/max-downscaling-image.html]: colors

File: public/demos/max-downscaling-image.html [hunk 6]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
+    </p>
+  </div>
+
+  <div class="layout vertical center-center">
+    <img src="../img/emilio-large.jpg" width="60" height="500">
+    <p>
+    Intrinsic size: 800px x 500px <br />
+    Container size: 60px x 500px
+    For devices with device pixel ratio between 3 to 5 (e.g. iPhoneX, LG G3+, Samsung Galaxy S4, etc),
+    the compression raio (800 / (60 * 5)) > 2, image renders with inverted clolors.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: colors

File: public/demos/max-downscaling-image.html [hunk 7]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
+    For devices with device pixel ratio of 2 (e.g. iPad, MacBook, smartphones, etc), the compression ratio (800/(180 * 2)) > 2, image renders with inverted clolors.
+    </p>
+  </div>
+
+  <div class="layout vertical center-center">
+    <img src="../img/emilio-large.jpg" width="60" height="500">
+    <p>
+    Intrinsic size: 800px x 500px <br />
+    Container size: 60px x 500px
+    For devices with device pixel ratio between 3 to 5 (e.g. iPhoneX, LG G3+, Samsung Galaxy S4, etc),
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: image renders -> the image renders

File: public/demos/max-downscaling-image.html [hunk 8]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
+    the compression raio (800 / (60 * 5)) > 2, image renders with inverted clolors.
+    </p>
+  </div>
+</section>
+<section class="layout center wrap" style="justify-content:space-evenly;">
+  <h3>
+    When the compression ratio (height) is more than maximum downscaling raio, image renders
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: image renders -> the image renders

File: public/demos/max-downscaling-image.html [hunk 9]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
+    with inverted colors.
+  </h3>
+  <div class="layout vertical center-center">
+    <img src="../img/emilio-large.jpg" width="800" height="500">
+    <p>
+    Intrinsic size: 800px x 500px <br />
+    Container size: 800px x 200px
+    For devices with device pixel ratio of 1 (e.g. tablets, monitors, etc), the compression ratio (500/200) > 2, image renders with inverted clolors.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: image renders -> the image renders

Comment (ebidel) [reply public/demos/max-downscaling-image.html]: colors

File: public/demos/max-downscaling-image.html [hunk 10]
```diff
@@ -34,35 +34,114 @@ <h2 id="feature-allowed-banner">
...
 
   <div class="layout vertical center-center">
-    <img src="../img/Fisher-large-stripped.jpg" width="80" height="150">
+    <img src="../img/emilio-large.jpg" width="800" height="100">
     <p>
-    Intrinsic size: 200px x 150px <br />
-    Container size: 80px x 150px
+    Intrinsic size: 800px x 500px <br />
+    Container size: 800px x 100px
+    For devices with device pixel ratio of 2 (e.g. iPad, MacBook, smartphones, etc), the compression ratio (800/(200 * 2)) > 2, image renders with inverted clolors.
```
Comment (ebidel) [inline public/demos/max-downscaling-image.html]: same on all these

**Comment by loonybear** [comment]
Done. Please take another look. Thanks.
I also updated the policy name with the most recent change.

**Pull request merged**